webpack/scenes/Hosts/ChangeContentSource/components/ContentSourceTemplate.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import {
+  Grid,
+  GridItem,
+  CodeBlock,
+  CodeBlockAction,
+  CodeBlockCode,
+  ClipboardCopyButton,
+  ExpandableSection,
+  ExpandableSectionToggle,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+
+import { copyToClipboard } from '../helpers';
+
+const ContentSourceTemplate = ({ template, jobInvocationPath }) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isCopied, setCopied] = useState(false);
+
+  const handleCopy = (e) => {
+    copyToClipboard(e, template);
+    setCopied(true);
+  };
+
+  const actions = () => (
```

Comment:
No need for this to be a function. you can just say
```suggestion
  const actions = (
```